kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/integrations/IdController.java

```diff
@@ -8,12 +8,27 @@
 
 import org.apache.commons.lang3.StringUtils;
 
+/**
+ * Controller for ids
+ * 
+ * @author Antti Leppä
+ */
 @Dependent
 public class IdController {
   
+  private IdController() {
+  }
+  
   @Inject
   private Instance<IdProvider> idProviders;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)